connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResource.java

```diff
@@ -78,7 +78,9 @@ public ConfigInfos validateConfigs(
             );
         }
 
-        return herder.validateConnectorConfig(connectorConfig);
+        return herder.validateConnectorConfig(connectorConfig,
+            // the validated configs don't need to be logger.
```

Comment:
A per argument inline comment is not something I've seen being used widely in Connect's codebase.